src/urllib3/filepost.pyi

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Generator, List, Mapping, Optional, Tuple, Union
+
+from . import fields
+
+Fields = Union[Mapping[str, str], List[Tuple[str]]]
+Iterator = Generator[Tuple[str], None, None]
+
+RequestField = fields.RequestField
```

Comment:
`RequestField` is unused, but it looks like `iter_field_objects` and `encode_multipart_formdata` should accept this type too?

Reply:
@pquentin Good catch, amended.